### ISSUE TITLE
No Update is happening. Status and resolution not set when issue is fixed 

### ISSUE THREAD

**Issue by MimirRnD** [mantisbt-plugins/source-integration#332]
I have installed the plug in to Mantis but I do not get it right.
I add "resolved #146" in the commits on the CLI  but the status is not updated to the resolved as we have as our next status.

Please see screen shots for configuration

When I test this it does not work as it should

What is wrong here. Please advise

![Annotation 2020-01-01 121606](https://user-images.githubusercontent.com/29481072/71641252-8e262a00-2c90-11ea-9d3f-da3ef858715e.png)
![Annotation 2020-01-01 121550](https://user-images.githubusercontent.com/29481072/71641253-8ebec080-2c90-11ea-8a04-5890fc60afd5.png)

**Comment by steffen-wilke** [comment]
I'm currently experiencing the same issue. The changeset is attached, the "Bug Fixed Message" is created, but the issue itself is not set to resolved.

**Comment by jchampeau** [comment]
I encountered the a similar issue.  Changesets were being added to the bug but the bug was not being marked as fixed/resovled.  I found 2 issues:

1. The configuration option "Bug Fixed Assign To Committer" needed to be enabled.  I was confused by wording as described in [#229](https://github.com/mantisbt-plugins/source-integration/issues/229).  

2. The SVN username did not match to any username in MantisBT.  Once I corrected this it started working for me.

While trying to figure out the second problem, I was looking at the function Source_Process_Changesets() in Source.API.php.  There are checks on the user existing and having enough privileges to update the status/resolution fields.  I added some logging to figure out which conditions were preventing the updating of the bug's status.  There are also some checks on status thresholds.  So if you have customized your statuses maybe that could be causing issues.
